### PR TITLE
fix(Vulnerable dependency): dependency version update for jquery and bootstrap

### PR DESCRIPTION
**Pull request by VisaLilyFeng**
### SUMMARY
1, The application is running a vulnerable version of jQuery, v3.4.1, which was released on May 1, 2019.
https://snyk.io/test/npm/jquery/3.4.1
Impact
The noted version of the product is vulnerable to cross-site scripting.

2, The application is running a vulnerable version of Bootstrap v3.4.1, which was released on February 15, 2019.

Impact
The noted version of the product is vulnerable to cross-site scripting attacks. However this vulnerability was not identified during this test and the finding is based on the vulnerable version returned, hence warrants a low severity rating.

CVE ID:
• CVE-2019-8331

### CODE REVIEW

File: superset-frontend/package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "antd": "^4.9.4",
     "array-move": "^2.2.1",
     "babel-plugin-typescript-to-proptypes": "^2.0.0",
-    "bootstrap": "^3.4.1",
+    "bootstrap": "^5.1.1",
```
Comment (etr2460) [inline superset-frontend/package.json:116]: bootstrap is only used to enable menus from FAB to work. So 2 questions:
1. Do we still need this, or have we replaced the FAB menus everywhere?
2. Do we need to ensure FAB is using bootstrap v5 before we make this upgrade?

Comment (etr2460) [reply superset-frontend/package.json:116]: cc @dpgaspar

Comment (etr2460) [reply superset-frontend/package.json:116]: The examples are using 4.2.1, perhaps we need that here too: https://github.com/dpgaspar/Flask-AppBuilder/blob/27b15e59316e85e0fe62b8aa9978391ed4c729c9/examples/react-rest-api/app/static/package.json#L7

Not sure if that has the vuln fixed in it or not

Comment (VisaLilyFeng) [reply superset-frontend/package.json:116]: > The examples are using 4.2.1, perhaps we need that here too: https://github.com/dpgaspar/Flask-AppBuilder/blob/27b15e59316e85e0fe62b8aa9978391ed4c729c9/examples/react-rest-api/app/static/package.json#L7
> 
> Not sure if that has the vuln fixed in it or not
According to  CVE-2019-8331 the issue fixed in version 4.3.1

Comment (nytai) [reply superset-frontend/package.json:116]: We removed bootstrap from the frontend, are we exposing this package anywhere or just using the styles? I only see references to it in `package.*.json`

Comment (VisaLilyFeng) [reply superset-frontend/package.json:116]: @etr2460 If we still use it, can we bump up the version to a secure version? For both jquery and bootstrap?

**Comment by rusackas** [comment]
@eschutho @michael-s-molina @dpgaspar  Just pointing out this fairly old PR is still open... I'm not sure if it can be easily made mergeable, or if we should open a new/rebased one, but it does underscore that the move to a React Single Page app is not just a feature and maintenance issue, but a security one as well.

**Comment by rusackas** [comment]
I merged a couple of PRs, one removing bootstrap, and one bumping jquery. I think we're all good here now. Thanks for the contribution, and nag me on [slack](http://bit.ly/join-superset-slack) if you ever open more that need attention :D

**Pull request closed without merging**